.github/workflows/dapr.yml

```diff
@@ -285,66 +346,63 @@ jobs:
+      - name: "download artifact: dapr_linux_amd64_all"
+        uses: actions/download-artifact@master
+        with:
+          name: dapr_linux_amd64_all
+          path: ${{ env.ARTIFACT_DIR }}
+      - name: "download artifact: dapr_linux_amd64_stable"
         uses: actions/download-artifact@master
         with:
-          name: dapr_linux_amd64
+          name: dapr_linux_amd64_stable
           path: ${{ env.ARTIFACT_DIR }}
-      - name: "download artifact: dapr_linux_arm"
+      - name: "download artifact: dapr_linux_arm_all"
         uses: actions/download-artifact@master
         with:
-          name: dapr_linux_arm
+          name: dapr_linux_arm_all
           path: ${{ env.ARTIFACT_DIR }}
-      - name: "download artifact: dapr_linux_arm64"
+      - name: "download artifact: dapr_linux_arm_stable"
         uses: actions/download-artifact@master
         with:
-          name: dapr_linux_arm64
+          name: dapr_linux_arm_stable
           path: ${{ env.ARTIFACT_DIR }}
-      - name: "download artifact: dapr_windows_amd64"
+      - name: "download artifact: dapr_linux_arm64_all"
         uses: actions/download-artifact@master
         with:
-          name: dapr_windows_amd64
+          name: dapr_linux_arm64_all
           path: ${{ env.ARTIFACT_DIR }}
-      - name: "download artifact: dapr_darwin_amd64"
+      - name: "download artifact: dapr_linux_arm64_stable"
         uses: actions/download-artifact@master
         with:
-          name: dapr_darwin_amd64
+          name: dapr_linux_arm64_stable
           path: ${{ env.ARTIFACT_DIR }}
-      - name: "download artifact: dapr_darwin_arm64"
+      - name: "download artifact: dapr_windows_amd64_all"
         uses: actions/download-artifact@master
         with:
-          name: dapr_darwin_arm64
+          name: dapr_windows_amd64_all
+          path: ${{ env.ARTIFACT_DIR }}
+      - name: "download artifact: dapr_darwin_amd64_all"
+        uses: actions/download-artifact@master
+        with:
+          name: dapr_darwin_amd64_all
+          path: ${{ env.ARTIFACT_DIR }}
+      - name: "download artifact: dapr_darwin_arm64_all"
+        uses: actions/download-artifact@master
+        with:
+          name: dapr_darwin_arm64_all
           path: ${{ env.ARTIFACT_DIR }}
       - name: "download artifact: dapr_grafana_dashboards"
         uses: actions/download-artifact@master
         with:
           name: dapr_grafana_dashboards
           path: ${{ env.ARTIFACT_DIR }}
-      - name: tar release binaries
+      - name: list tar release binaries
         run: |
           IMAGE_ARCHS=(amd64 arm arm64)
           for img_arch in "${IMAGE_ARCHS[@]}"; do
-            echo "Preparing linux_${img_arch} image..."
-            RELEASE_DIR=dist/linux_${img_arch}/release/
-            mkdir -p $RELEASE_DIR
-            for tgzfile in $ARTIFACT_DIR/*linux_${img_arch}.tar.gz; do
-              echo "Extracting $tgzfile ..."
-              tar xzvf $tgzfile -C $RELEASE_DIR
```

Comment:
The output of this step was not being used anywhere, so I am changing it to just print it out.